helpers/test-helper.js

```diff
@@ -180,6 +180,10 @@ export async function forwardTime(seconds, test) {
   return p;
 }
 
+export function getFunctionSignature(sig) {
+  return web3Utils.sha3(sig).slice(0, 10);
```

Comment:
There must be a function in one of the packages we import that already does that. Like `ethers` or https://web3js.readthedocs.io/en/1.0/web3-eth-abi.html#encodefunctionsignature

Reply:
The only package we have is `web3-utils` which doesn't have `encodeFunctionSignature`. I just thought that maybe we shouldn't add whole library for one function, except if we are going to use it later

Reply:
The only other option is the `Resolver.stringToSig` function but let's leave it as it is for now.